tools/bvgraph2smat/bvgraph2smat.cc

```diff
@@ -82,7 +82,7 @@ bool write_badj_format(bvgraph *g, bvgraph_iterator *git, const char *outname) {
     fwrite(&n, sizeof(int64_t), 1, f);
     fwrite(&m, sizeof(int64_t), 1, f);
     
-    if (m > 4294967296)
+    if (n > 4294967296)
```

Comment:
You want if n > 4294967295 (2^31-1) or maybe even 4294967294 (2^32-2) because 2^31-1 is often used as a sentinal value ... will accept with this one change!